libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -81,6 +85,9 @@ void AP_LandingGear::deploy()
 
     // set deployed flag
     _deployed = true;
+
+    // GCS message & log
+    gcs().send_text(MAV_SEVERITY_INFO, "Landing Gear Deployed");
```

Comment:
I'd actually prefer if we don't send out too many send-text messages.  I think we should limit the send texts to when things go wrong.  The problem is that it assumes a link with a decent band width and that's not always the case.

Reply:
It is more for the purpose of recording the gear action in the log, which I don't believe is done by any other means.

Reply:
for logging in tlog or dataflash ? For dataflash, is it already logged in landinggear_update(). Even if I think it would be simplier that the library do it directly.

Reply:
I'll just delete this portion of it entirely.